.env

```diff
@@ -1,3 +1,4 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
@CalDev98 what's this setting for? Why should we add it

Reply:
> @CalDev98 what's this setting for? Why should we add it

I just used it because of my webpack version